### PR TITLE
Fix issue #288 - added implementation of duplicatesAndUnique

### PR DESCRIPTION
**Pull request by AnujaP2020**
Added implementations of duplicatesAndUnique method in haiku-kata-solutions and updated the test case for the same.

### CODE REVIEW

File: haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorEC.java [hunk 1]
```diff
@@ -60,18 +60,17 @@ public String distinctLetters()
...
-        // Hint: Find a method on MutableCharBag that returns unique chars
-        CharSet unique = null;
+        // Find all of the alphabetic letters, convert them to lowercase
+        MutableCharBag chars = this.getHaikuAsCharAdapter()
+                                .select(Character::isAlphabetic)
+                                .collectChar(Character::toLowerCase)
+                                .toBag();
+
+        // Find all the chars with duplicates
+        CharBag duplicates = chars.selectByOccurences(occurences -> occurences > 1);
```
Comment (rzrobin213) [inline haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorEC.java:70]: there is a more concise way to do this :) look at the methods in  MutableCharBag.
https://eclipse.dev/collections/javadoc/10.4.0/org/eclipse/collections/api/bag/primitive/MutableCharBag.html

File: haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorEC.java [hunk 2]
```diff
@@ -60,18 +60,17 @@ public String distinctLetters()
...
+
+        // Find all the unique chars
+        CharSet unique = chars.selectByOccurences(occurences -> occurences == 1).toSet();
```
Comment (rzrobin213) [inline haiku-kata-solutions/src/main/java/org/eclipse/collections/haikukata/TextProcessorEC.java:73]: same as last comment

**Review by rzrobin213 — CHANGES_REQUESTED** [review]
left some comments

**Review by rzrobin213 — APPROVED** [review]
Looks good!

**Comment by rzrobin213** [comment]
All you need is to rebase and we can merge

**Review by prathasirisha — APPROVED** [review]

**Comment by prathasirisha** [comment]
Thank you for your contribution @AnujaP2020

**Pull request merged**